modules/apps/document-library/document-library-taglib/src/main/java/com/liferay/document/library/taglib/internal/display/context/RepositoryBrowserTagDisplayContext.java

```diff
@@ -527,7 +527,8 @@ private SearchContainer<Object> _getDLSearchContainer()
 				_httpServletRequest, searchContainer.getOrderByColParam()));
 		searchContainer.setOrderByType(
 			ParamUtil.getString(
```

Comment:
On the old jsp it put to default this too:

`String orderByCol = ParamUtil.getString(request, "orderByCol", "title");`